packages/extension-code-block/src/code-block.ts

```diff
@@ -248,7 +253,7 @@ export const CodeBlock = Node.create<CodeBlockOptions>({
 
             const currentLine = lines[currentLineIndex]
             const leadingSpaces = currentLine.match(/^ */)?.[0] || ''
-            const spacesToRemove = Math.min(leadingSpaces.length, this.options.tabSize)
+            const spacesToRemove = Math.min(leadingSpaces.length, this.options.tabSize || 4)
```

Comment:
Just a nitpick - I don't like how we re-use the `4` hardcoded here.

Maybe it makes sense to move this out into a constant `const DEFAULT_TAB_SIZE = 4` and use it in `addOptions` and all other occurences.

You could also do something like this:

```
const { tabSize = DEFAULT_TAB_SIZE } = this.options
```

a few lines before you use the value and then just use `tabSize`